src/plugins/discover/public/components/discover_grid/discover_grid.tsx

```diff
@@ -166,6 +166,10 @@ export interface DiscoverGridProps {
    * Update row height state
    */
   onUpdateRowHeight?: (rowHeight: number) => void;
+  /**
+   * Callback to execute on edit runtime field
+   */
+  onEditRuntimeField?: () => void;
```

Comment:
Since we are not only can edit runtime fields, but also add custom labels and formatting to existing 'normal' fields, I think we should just label it `onEditField`